rtrlib/bgpsec/bgpsec_utils.c

```diff
@@ -27,7 +27,17 @@ struct stream *init_stream(uint16_t size)
 {
 	struct stream *s = lrtr_calloc(sizeof(struct stream), 1);
```

Comment:
maybe while you are at it. 

please rename this variable, single character variables are a clear code smell.